docs/developers/SETUP_AWS_ENVIRONMENTS.md

```diff
@@ -0,0 +1,82 @@
+# Setup the developers AWS environments
+
+## Prerequisite
+
+* Have a Gitlab account that has been added to GigaScience project
+* Have a project under "Gigascience/forks" named "\<github handle\>-gigadb-website
+* Have the domain names registered and DNS A record pointing to the following EIPs (which also must be created):
+
+| endpoint | environment | domain | EIP | 
+| -- | -- | -- | 
```

Comment:
Need to add another column in order for markdown table to be properly rendered.

Reply:
fixed